src/elm/Page/Community.elm

```diff
@@ -482,6 +484,51 @@ viewAction loggedIn metadata maybeDate action =
             ]
 
 
+viewClaimModal : LoggedIn.Model -> Model -> Html Msg
+viewClaimModal loggedIn model =
+    case model.modalStatus of
+        Opened isLoading actionId ->
```

Comment:
Just a small thing. This should be `open` instead. `Opened` is used as the past tense of `open` (he opened the door). We use `open` when we want to denote `not closed`.

Reply:
nice! I'll fix it! small change but important